modules/cloudaccess/wf_privatelink_manager_definition.json

```diff
@@ -0,0 +1,21 @@
+{
+    "actions": [
+        "Microsoft.Authorization/roleAssignments/read",
+        "Microsoft.Authorization/roleDefinitions/read",
+        "Microsoft.Network/privateEndpoints/read",
+        "Microsoft.Network/privateEndpoints/write",
+        "Microsoft.Network/privateEndpoints/delete",
+        "Microsoft.Network/privateDnsZones/read",
+        "Microsoft.Network/privateDnsZones/write",
```

Comment:
Is `delete` not needed on this resource, for clean-up?

Reply:
It isn't - wayfinder doesn't ever clean up this private DNS zone (by design)